share/spice/cryptocurrency/cryptocurrency.js

```diff
@@ -0,0 +1,250 @@
+(function (env) {
+    "use strict";
+    
+    // Currencies that we have flags for.
+    // These correspond to traditional currencies of countries.
+    var currency2country = {
+        "aud": true,
+        "cny": true,
+        "eur": true,
+        "gbp": true,
+        "hkd": true,
+        "jpy": true,
+        "nzd": true,
+        "pln": true,
+        "rur": true,
+        "sgd": true,
+        "usd": true
+    };
+    
+    // Cryptocurrencies that we have flags for.
+    var crypto_flags = {
+        "btc": true,
+        "doge": true,
+        "drk": true,
+        "ltc": true,
+        "xrp": true
+    }
```

Comment:
Should you add the other cryptocurrencies in here now that you have more? Queries such as "convert ltc to qrk" will still show qrk with a default flag.